.github/workflows/releases.yml

```diff
@@ -25,29 +27,38 @@ jobs:
         env:
           GH_OAUTH_CLIENT_ID: 178c6fc778ccc68e1d6a
           GH_OAUTH_CLIENT_SECRET: ${{secrets.OAUTH_CLIENT_SECRET}}
-          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          GITHUB_TOKEN: ${{secrets.UPLOAD_GITHUB_TOKEN}}
   msi:
     needs: goreleaser
     runs-on: windows-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
-      - name: Set up Go 1.13
-        uses: actions/setup-go@v1
-        with:
-          go-version: 1.13
+        uses: actions/checkout@v2
       - name: Download gh.exe
-        uses: ./.github/actions/download-exe
         id: download_exe
+        shell: bash
+        run: |
+          curl -fsSL https://github.com/github/hub/raw/master/script/get | bash -s 2.14.1
+          bin/hub release download "${GITHUB_REF#refs/tags/}" -i '*windows_amd64*.zip'
+          printf "::set-output name=zip::%s\n" *.zip
+          unzip -o *.zip && rm -v *.zip
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
       - name: Install go-msi
         run: choco install -y "go-msi"
+      - name: Prepare PATH
+        shell: bash
+        run: |
+          echo "::add-path::$WIX\\bin"
+          echo "::add-path::C:\\Program Files\\go-msi"
       - name: Build MSI
         id: buildmsi
-        uses: ./.github/actions/build-msi
-        with:
-          exe: ${{ steps.download_exe.outputs.exe }}
+        shell: bash
+        run: |
+          mkdir -p build
+          msi="$(basename "${{ steps.download_exe.outputs.zip }}" ".zip").msi"
+          printf "::set-output name=msi::%s\n" "$msi"
+          go-msi make --msi "$PWD/$msi" --out "$PWD/build" --version "${GITHUB_REF#refs/tags/}"
```

Comment:
Now that the `download_exe` has changed and already extracts `bin/hub.exe` in the appropriate place, I've decided to inline the build MSI script since most of what it does is just invokes `go-msi` with appropriate arguments.